src/components/inputs/Dates/Calendar/Calendar.tsx

```diff
@@ -262,23 +279,33 @@ export const Calendar = ({
   );
 };
 
-const DAYS_OF_WEEK = {
-  Sunday: 'Su',
-  Monday: 'Mo',
-  Tuesday: 'Tu',
-  Wednesday: 'We',
-  Thursday: 'Th',
-  Friday: 'Fr',
-  Saturday: 'Sa',
-};
-
-const DAY_LABELS = Object.values(DAYS_OF_WEEK);
-
-function month(month) {
-  return month.toLocaleDateString('en-US', {
-    month: 'long',
-    year: 'numeric',
-  });
+function getMonthName(month: number) {
+  switch (month) {
+    case 0:
+      return 'january';
+    case 1:
+      return 'february';
+    case 2:
+      return 'march';
+    case 3:
+      return 'april';
+    case 4:
+      return 'may';
+    case 5:
+      return 'june';
+    case 6:
+      return 'july';
+    case 7:
+      return 'august';
+    case 8:
+      return 'september';
+    case 9:
+      return 'october';
+    case 10:
+      return 'november';
+    case 11:
+      return 'december';
+  }
```

Comment:
instead of this function we could also use

```
currentDate.toLocaleString('en', { month: 'long' }).toLowerCase();
```
to get the lowercase english months, but i prefer the switch. lmk if you want me to change it